src/components/Pagination/Pagination.tsx

```diff
@@ -135,7 +135,7 @@ const Pagination: React.FC<PaginationProps> = ({
         <Container>
             <ButtonsContainer>
                 {hasMultiplePageSizes && (
-                    <Box aria-label={ariaLabelSelectPageSizeContainer} position="absolute" left="0" width="4.5em">
+                    <Box aria-label={ariaLabelSelectPageSizeContainer} position="absolute" left="0">
```

Comment:
This is a bug that was [already fixed in v2 by using `minWidth="5em"`](https://github.com/freenowtech/wave/blob/main/src/components/Pagination/Pagination.tsx#L145), would the same fix work for DMT or do you need to remove the `width` altogether?

Reply:
Without having any context, is there particular reason you set `em` there, not `rem`?

Reply:
But yeah, we can use this fix instead for now to not cause overlapping of selector with pagination buttons 👍

Reply:
I don't recall the reasoning behind using `em` tbh 😅 I see you are using `rem` so I think it's fine